programmers/난이도별/level01.2016년/s2uyeoii.c

```diff
@@ -0,0 +1,20 @@
+#include <string>
+#include <vector>
+
+using namespace std;
+
+string weeks[7] = { "FRI","SAT","SUN","MON","TUE","WED","THU"};
+int month[12] = { 31,29,31,30,31,30,31,31,30,31,30,31 };
+
+string solution(int a, int b) {
+	string answer = "";
+	int day = 0;
+	for (int k = 0; k <(a-1); k++) {
+		day += month[k];
+	}
+	day += (b-1);
+	day %= 7;
+	answer = weeks[day];
+
+	return answer;
```

Comment:
```suggestion
	int day = b - 1;
	for (int k = 0; k <(a-1); k++) {
		day += month[k];
	}
	day %= 7;
	return weeks[day];
```

꼭 필요한 변수가 아니라면 선언 하지 않아도 될 것 같아요!

Reply:
C++에서 이렇게 써도 될지 모르겠지만 한 줄만 더 욕심을 낸다면
```
return weeks[day % 7];
```
로 바꿀수도 있겠네요.